stmt_go19.go

```diff
@@ -1,13 +1,105 @@
+// +build go1.9
```

Comment:
This is probably a typo ? 

```suggestion
// +build go1.19
```

Reply:
No? Go 1.9 added the NamedValueChecker interface if I'm not mistaken. Go 1.19 doesn't exist yet?

Reply:
Oh! I wasn't very awake when I wrote this, clearly, oops